glui_textbox.cpp

```diff
@@ -180,8 +180,9 @@ int    GLUI_TextBox::key_handler( unsigned char key,int modifiers )
   /*  has_selection = (sel_start != sel_end);              */
 
   if ( key  == CTRL('[')) {         /* ESCAPE */
-    glui->deactivate_current_control();
-    return true;
+    
+	  //GLUI_Main::keyboard returns control to the container.
+	  assert(0); 
```

Comment:
BTW: While this is the "TextBox" the only reason I rewrote a lot of the EditText key_handler is because there were bugs causing the selection and insertion_pt and text.size properties to disagree and crashing.